incubator/beast/templates/configmap.yaml

```diff
@@ -5,33 +5,46 @@ metadata:
   namespace: {{ .Values.namespace }}
 data:
   BQ_DATASET_NAME: "{{ .Values.configArgs.bqDatasetName }}"
+  BQ_PROJECT_NAME: "{{ .Values.configArgs.bqProjectName }}"
   BQ_TABLE_NAME: "{{ .Values.configArgs.bqTableName }}"
+  BQ_TABLE_PARTITION_KEY: "{{ .Values.configArgs.bqTablePartitionKey }}"
   BQ_WORKER_POOL_SIZE: "{{ .Values.configArgs.bqWorkerPoolSize }}"
-  GOOGLE_CREDENTIALS: "{{ .Values.configArgs.googleCredentials }}"
+  BQ_WORKER_POLL_TIMEOUT_MS: "{{ .Values.configArgs.bqWorkerPollTimeoutMs | default 5000 }}"
+  ENABLE_BQ_TABLE_PARTITIONING: "{{ .Values.configArgs.enableBqTablePartitioning }}"
```

Comment:
Only if `ENABLE_BQ_TABLE_PARTITIONING` is true, then set `BQ_TABLE_PARTITION_KEY`,also `BQ_TABLE_PARTITION_KEY` should be required if `ENABLE_BQ_TABLE_PARTITIONING` is true